traefik/mesh
# Add HTTP port mapper

## What does this PR do?

This PR:
- Adds an HTTP port mapper to mimic how mapping is handled for TCP and UDP.
- Refactor the shadow service manager
- Makes port mappers idempotent

### How to test it

* `make test`
* Create services, update them and remove them. Make sure the shadow services stay in sync.
